src/input/mpegts/satip/satip.c

```diff
@@ -508,6 +525,12 @@ satip_device_create( satip_device_info_t *info )
     }
   }
 
+  if (sd->sd_fritz_quirk == 1) {
```

Comment:
I meant - try to name the quirk using an unversal name and do only the "default" settings when the hw is detected. You're trying to override other settings when sd->sd_fritz_quirk is set by user which may not be wanted.